internal/extramanifest/extramanifest.go

```diff
@@ -47,18 +47,26 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+//+kubebuilder:rbac:groups="",resources=namespaces,verbs=get;watch;create;
+
 const (
-	ExtraManifestPath           = "/opt/extra-manifests"
-	PolicyManifestPath          = "/opt/policy-manifests"
-	WarnAnnotationUnknownCRDKey = "lca.openshift.io/warn-extramanifest-cm-unknown-crd"
+	ExtraManifestPath                        = "/opt/extra-manifests"
+	PolicyManifestPath                       = "/opt/policy-manifests"
+	ExtraManifestValidationWarningAnnotation = "lca.openshift.io/extramanifest-cm-validation-warning"
```

Comment:
```suggestion
	Path                        = "/opt/extra-manifests"
	PolicyManifestPath                       = "/opt/policy-manifests"
	ValidationWarningAnnotation = "lca.openshift.io/extramanifest-cm-validation-warning"
```
optional nit: exported constants should not start with package name (see [styleguide](https://google.github.io/styleguide/go/decisions#constant-names))

in this way we could call it like `extramanifest.ValidationWarningAnnotation` instead of `extramanifest.ExtraManifestValidationWarningAnnotation` (repeated)

totally get it if we don't want to go over this refact in this PR, just suggesting it